core/src/main/java/io/questdb/cutlass/pgwire/modern/PGConnectionContextModern.java

```diff
@@ -143,6 +144,7 @@ public class PGConnectionContextModern extends IOContext<PGConnectionContextMode
     private static final int PREFIXED_MESSAGE_HEADER_LEN = 5;
     private static final int PROTOCOL_TAIL_COMMAND_LENGTH = 64;
     private static final int SSL_REQUEST = 80877103;
+    private static final long TIMEOUT_MILLIS = 5000; // 5 seconds timeout
```

Comment:
nit: can we please rename this constant to reflect what timeout is for. This is not the only timeout in the system.

Reply:
also, in the comment, instead of stating the obvious, the comment should explain why this value is there at all

Reply:
![image](https://github.com/user-attachments/assets/c4e0026b-acf9-4f62-8f0a-ea840881634c)

Reply:
addressed